src/modules/dashboard/components/CoinMachine/BuyTokens/BuyTokens.tsx

```diff
@@ -397,7 +402,7 @@ const BuyTokens = ({ colony: { colonyAddress }, disabled }: Props) => {
                       <FormattedMessage {...MSG.costLabel} />
                     </div>
                     <div className={styles.amountsValues}>
-                      {!disabled ? (
+                      {isCurrentlyOnSale ? (
```

Comment:
and probably here,  unless we alrady in "else"

Reply:
good point in all 3 cases.